src/plugins/embeddable/public/embeddable_panel/embeddable_panel.tsx

```diff
@@ -55,9 +60,20 @@ const getComponentFromEmbeddable = async (
 
 /**
  * Loads and renders a legacy embeddable.
+ *
+ * Ancestry chain must use 'key' attribute to reset DOM and state when embeddable changes
```

Comment:
Wondering why we call this `Ancestry chain` here instead of parent component?

Reply:
Because in the case of implementation it looks like the below, so its not the direct parent with the key attribute.

```
<DashboardGridItem key={id}>
  <Item>
    <EmbeddablePanel>
```

Reply:
Makes sense, thanks for the explanation!